php/blocks/migration/class-post-type.php

```diff
@@ -143,4 +119,20 @@ public function migrate_single( WP_Post $post ) {
 
 		return ! empty( $rows_updated );
 	}
+
+	/**
+	 * Gets the posts of the previous post_type.
+	 *
+	 * @return WP_Post[] The posts that were found.
+	 */
+	private function query_for_posts() {
+		$query = new WP_Query(
+			[
+				'post_type'      => $this->previous_post_type_slug,
+				'posts_per_page' => 10,
```

Comment:
I don't understand how this can work without an offset parameter?

Reply:
Yeah, this is confusing.

When the posts are migrated, they have their `post_type` changed. So this `WP_Query` won't find posts that were migrated. 

When this did have an `offset`, the offset eventually got high enough that there were no posts.

Reply:
Ahhh – that makes total sense. Maybe leave a comment in there to that effect?

Reply:
Sure, good idea. I missed that in the first iteration of this 😄

Reply:
https://github.com/getblocklab/block-lab/pull/588/commits/3a3afefc054316cb48c20977b6b1c2f99cae1ba9 adds a comment about that